pkg/manifests/manifests.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/openshift/cluster-monitoring-operator/pkg/promqlgen"
 	"github.com/pkg/errors"
 	monv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"golang.org/x/crypto/bcrypt"
```

Comment:
This change is covered in https://github.com/openshift/cluster-monitoring-operator/pull/1363/files#diff-1f9d2d18d4cabd8262e585b932330183f7cb9b544ac9f27c8bcb3ef9c8d14376R34   :)

Reply:
Good catch. 
I don't think there is a problem with having this change in two PRs, it should not cause any conflicts. 
Also, I would like to enable go linting checks in the CI and for that PR to be merged, the codebase has to be properly formatted.